packages/map-template/src/components/MapTemplate/MapTemplate.jsx

```diff
@@ -544,6 +547,20 @@ function MapTemplate({ apiKey, gmApiKey, mapboxAccessToken, venue, locationId, p
         setCategories(uniqueCategories);
     }
 
+    /*
+     * React on changes in the category prop.
+     * Check if the category property matches with any of the existing categories.
+     */
+    useEffect(() => {
+        if (mapsindoorsSDKAvailable) {
+            if (category && categories.includes(matched => matched[0] === category) {
```

Comment:
The code cannot compile due to a syntax error here. When I say my suggestions are not tested, I really mean it 😅

Reply:
Reverted the changes 👍